gulpfile.js

```diff
@@ -247,6 +250,23 @@ gulp.task("link-bugs", function (done) {
     exec(bin("linkify-bugs.sh"), [path.join(SOURCE_DIR, "_posts")], done);
 });
 
+gulp.task('lint', function() {
+    return gulp.src(path.join(LINT_DIR, "**", "*.html"))
+        .pipe(htmllint());
+});
+
+gulp.task('checklinks', function(cb) {
+  crawler.crawl('http://localhost:3000/')
```

Comment:
This gulp task should depend on `serve` task. 
Nit: Also, the URL should be set for browser-link and used here as a constant - or get the default from browser-link if that's exposed instead of hardcoding this here.